### PR TITLE
[16.0][MIG] storage_thumbnail

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Standard Migration

@ForgeFlow

### CODE REVIEW

File: storage_thumbnail/models/storage_thumbnail.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        """Choose the correct backend.
+
+        By default : it's the one configured as ir.config_parameter
+        Overload this method if you need something more powerfull
+        """
+        return self.env["storage.backend"]._get_backend_id_from_param(
+            self.env, "storage.thumbnail.backend_id"
+        )
+
+    @api.model
```
Comment (benwillig) [inline storage_thumbnail/models/storage_thumbnail.py]: we should use the api.model_create_multi here

File: storage_thumbnail/models/thumbnail_mixin.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+            self.thumbnail_ids.invalidate_recordset()
+        return thumbnail
+
+    def generate_odoo_thumbnail(self):
+        self_sudo = self.sudo()
+        self_sudo._get_small_thumbnail()
+        self_sudo._get_medium_thumbnail()
+        return True
+
+    @api.model
```
Comment (benwillig) [inline storage_thumbnail/models/thumbnail_mixin.py]: Also api.model_create_multi

File: storage_thumbnail/tests/models.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    alt_name = fields.Char(string="Alt Image name")
+    file_id = fields.Many2one("storage.file", "File", required=True, ondelete="cascade")
+
+    @api.model
+    def _get_backend_id(self):
+        return int(
+            self.env["ir.config_parameter"].get_param("storage.thumbnail.backend_id")
+        )
+
+    @api.model
```
Comment (benwillig) [inline storage_thumbnail/tests/models.py]: Also api.model_create_multi

**Review by benwillig — CHANGES_REQUESTED** [review]
Some minor changes, functional tests are ok

**Comment by JasminSForgeFlow** [comment]
> Some minor changes, functional tests are ok

All changes have been updated.

Thanks

**Comment by hspl-hanak** [comment]
Thanks for the migration @JasminSForgeFlow 
Tested and seems ok @sebastienbeau can we merge this ?

**Review by hspl-hanak — APPROVED** [review]

**Review by hspl-priva — APPROVED** [review]

**Comment by hspl-priva** [comment]
LGTM @JasminSForgeFlow
 @sebastienbeau when can we merge this ?

**Pull request closed without merging**